docs/dns-minikube.md

```diff
@@ -0,0 +1,170 @@
+# Federation DNS for Ingress and Service
+This tutorial describes how to set up a federation cluster DNS with [ExternalDNS](https://github.com/kubernetes-incubator/external-dns/) based on [CoreDNS](https://github.com/coredns/coredns) in [minikube](https://github.com/kubernetes/minikube) clusters. You need to:
+- Install ExternalDNS with etcd enabled CoreDNS as a provider
+- To support Ingress, enable [ingress controller](https://github.com/kubernetes/ingress-nginx) for your minikube clusters
+- To support Service, install [metallb](https://github.com/google/metallb) for your minikube clusters  
+
+You can enable either Ingress or Service, or both of them in your clusters.
+For related conceptions of Muilti-cluster Ingress and Service, you can refer to [ingressdns-with-externaldns.md](https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/ingressdns-with-externaldns.md) and [servicedns-with-externaldns.md](https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/servicedns-with-externaldns.md).
+
+## Creating federation cluster
+Install Federation-v2 with minikube in [User Guide](https://github.com/kubernetes-sigs/federation-v2/blob/master/docs/userguide.md)
+
+## Installing ExternalDNS
+Install ExternalDNS with CoreDNS as backend in your host cluster. You can follow the [tutorial](https://github.com/kubernetes-incubator/external-dns/blob/master/docs/tutorials/coredns.md)
+
+To make it work for federation resources, you need to use below ExternalDNS deployment instead of the tutorial one.  
+**Note**: You should replace value of ETCD_URLS with your own etcd client service IP address.
+```
+$ cat external-dns.yaml
```

Comment:
Can you please use command as following instead?

```shell
cat <<EOF | kubectl create -f -
...
EOF
```

Make sure adding `shell` for "```"